src/current/_includes/releases/v22.2/v22.2.15.md

```diff
@@ -35,7 +35,6 @@ Release Date: October 23, 2023
 - When upgrading from previous releases, it was possible that ambiguous references to [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) would fail to be resolved by ID if they were in the same schema. Additionally, the logging is improved for ambiguous cases where the reference cannot be resolved. [#110721][#110721]
 - Fixed a bug where [`RESET `](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#reset-a-storage-parameter-to-its-default-value)[(`ttl_expire_after`)](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expire_after) could incorrectly remove [`ttl_expiration_expression`](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expiration_expression). [#110750][#110750]
 - Fixed a bug where [`format_type`](https://www.cockroachlabs.com/docs/v22.2/functions-and-operators#compatibility-functions) builtin did not honor `typemod` information for [array](https://www.cockroachlabs.com/docs/v22.2/array) types, leading to incorrect output. [#110939][#110939]
-- External connection URLs now accept the scheme `azure-blob` for connections to [Azure](https://www.cockroachlabs.com/docs/v22.2/use-cloud-storage#url-format) Blob Storage. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage. [#111286][#111286]
```

Comment:
Removing from here so that users of v22.2 do not use an external connection holding an `azure-blob` scheme expecting it to work with changefeeds.